include/boost/json/basic_parser_impl.hpp

```diff
@@ -1202,6 +1240,7 @@ parse_escaped(
     int digit;
     char c;
     cs.clip(temp.max_size());
+    bool replaced_bad_utf16 = false;
```

Comment:
Shouldn't this variable be a data member of the parser, so that suspensions doesn't lose its state?

Reply:
Added as a data member to the parser